test/runtests.jl

```diff
@@ -113,6 +114,8 @@ using WeakRefStrings
     @test tryparsenext(Quoted(String,'"', '\\'), "\"\tx\"") |> unwrap == ("\tx", 5)
     opts = LocalOpts(',', true, '"', '\\', false, false)
     @test tryparsenext(StringToken(String), "x y",1,3, opts) |> unwrap == ("x", 2)
+
+    @test tryparsenext(StringToken(String), "abcβ") |> unwrap == ("abcβ", 6)
```

Comment:
wait this wasn't working?

Reply:
No, it didn't, because it took [this](https://github.com/JuliaComputing/TextParse.jl/blob/68d35e91c9449e422d6a3f85dab6b838046a593f/src/field.jl#L365) code path and that expects a valid character index for ``j``.